js/apps/system/_admin/aardvark/APP/react/.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
+    "trailingComma": "none",
+    "tabWidth": 2,
+    "printWidth": 80,
+    "useTabs": false,
+    "semi": true,
+    "singleQuote": false,
+    "bracketSpacing": true,
+    "arrowParens": "avoid",
+    "jsxSingleQuote": false,
+    "endOfLine": "lf",
+    "jsxBracketSameLine": false
+}
```

Comment:
Agree, but I just wanted to avoid a big diff for now. We need to pin a prettier version and then we can switch to defaults. Removed most of these, only keeping `trailingComma` & `arrowParens` to avoid a big diff